src/components/Main/Modal/SettingModal/ThemeTab/EditTheme.vue

```diff
@@ -127,29 +131,25 @@ export default defineComponent({
   emits: {
     'change-theme': (theme: Theme) => true
   },
-  setup(props, context: SetupContext) {
+  setup(
+    props,
+    context: SetupContext<{ 'change-theme': (theme: Theme) => boolean }>
```

Comment:
```suggestion
    context
```
`emits`を指定すればここは型推論でうまくいくので逆にかかないほうがいいです 🙇